docs/source/configuration/parameters.md

```diff
@@ -169,3 +169,6 @@ kedro run --params="key1=value with spaces,key2=value"
 ```
 
 Since key-value pairs are split on the first equals sign, values can contain equals signs, but keys cannot.
+
+
+> **Note:** If you want to override values of certain keys in your configuration with runtime parameters provided through the CLI option, you can use the [OmegaConfigLoader `runtime_params` resolver](advanced_configuration.md#how-to-override-configuration-with-runtime-parameters-with-the-omegaconfigloader).
```

Comment:
You can use the `{note}` notation instead, to maintain consistency. Eg - https://github.com/kedro-org/kedro/blob/63d7516760b05edad3fbfc21277fb9741ecd83de/docs/source/configuration/parameters.md?plain=1#L71-L73

Reply:
@ankatiyar, thanks! I agree, but I tried that syntax, and it doesn’t support links. Since I have one link there, my suggestion is to remove the word "Note" and keep the current syntax. What do you think?